tests/__mocks__/handlers.ts

```diff
@@ -1,11 +1,93 @@
+/* eslint-disable @typescript-eslint/naming-convention */
```

Comment:
Why? Can't you just write the code to be in line with our styles?

Reply:
Yeah it just means casting `issue_number` and `pull_number`